apps/trading/.eslintrc.json

```diff
@@ -19,7 +19,15 @@
     },
     {
       "files": ["*.js", "*.jsx"],
-      "rules": {}
+      "rules": {
+        "unicorn/filename-case": [
+          "error",
+          {
+            "case": "kebabCase",
+            "ignore": ["react-singleton-hook/**/*.js"]
```

Comment:
chore (blocking): This doesn't support globs 😢  https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/rules/filename-case.js#L155-L159